### DISCUSSION TITLE
In M2O inputs, instead of opening the side panel to select relation, search and choose the record in the input

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13978] [Feature Requests]
When creating/editing an item, the majority of the input is done in the form input itself. Opening the side panel for selecting a relation feels weird when the data could be taken from what's typed into the input to then query the collection and select the result from, (so the input is first used to get the search query, for which results are shown in a drop down). Like select2:

![Screenshot from 2022-06-18 17-38-00](https://user-images.githubusercontent.com/95102939/174448240-be0ecd0c-87b7-4ed2-b8b9-b92d2938e2ec.png)

![Screenshot from 2022-06-18 17-40-43](https://user-images.githubusercontent.com/95102939/174448334-ed9d03af-50a1-466d-9f10-5f5c3e5cffac.png)

Maybe the admin could set the field type to either be a select like this, or open the side panel?

Maybe this could also be uses in M2M with checkboxes for the selections. Select2 can even limit the amount of records chosen for a multiple selection

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
